### DISCUSSION TITLE
Get HTTP Sender Variable to Next Destination

### DISCUSSION THREAD

**Discussion by LLCJK** [nextgenhealthcare/connect#4664] [Q&A]
I have an HTTP Sender destination that runs an api call to get a CCD based on what the user inputs in to the channel. What do I put in the response in order to get the CCD down to the next destination?

**Comment by smetz02** [comment]
In the response transformer, the HTTP response is available in the "msg" variable.  You can use channelMap.put to store the variable.  You can then reference the variable in your next destination.

**Reply by tonygermano** [reply]
You do not need to do that. The previous destination's Response is already stored in the responseMap. You'd only be making a second copy of that, and CCDs are typically large enough that you don't want extra copies floating around in memory.
